sync-core/src/main/java/com/cloudant/sync/query/IndexCreator.java

```diff
@@ -281,4 +323,17 @@ private String createIndexIndexStatementForIndexName(String indexName, List<Stri
         return String.format("CREATE INDEX %s ON %s ( %s )", sqlIndexName, tableName, cols);
     }
 
+    private String createVirtualTableStatementForIndex(Index index,
+                                                       List<String> columns,
+                                                       List<String> indexSettings) {
+        String tableName = IndexManager.tableNameForIndex(index.indexName);
+        Joiner joiner = Joiner.on(",").skipNulls();
+        String cols = joiner.join(columns);
+        String settings = joiner.join(indexSettings);
+
+        return String.format("CREATE VIRTUAL TABLE %s USING FTS4 ( %s, %s )", tableName,
+                                                                              cols,
+                                                                              settings);
```

Comment:
Short answer:  The statement will fail with a `Error: unrecognized parameter: "tokenize=simple"`.

Longer answer:  Apparently, the `=` sign cannot be included in the column name.  It seems to trigger SQLite to think that you are passing in a parameter and value.  But since we are surrounding all of our columns with double quotes we never run into a situation where the parameter will be valid.  Like in the case of `"tokenize=simple"`, SQLite reads the parameter as `"tokenize` and the  value as `simple"`.  Hence the error message.  So really the only negative here is that we cannot honor `=` in our text index field names.